packs/github/config.schema.yaml

```diff
@@ -0,0 +1,52 @@
+---
+token:
+  description: "GitHub oAuth Token"
+  type: "string"
+  default: "{{user.github_enterprise_oauth_token}}"
```

Comment:
Turns out I got this totally wrong :-( and forgot to correct it...

I've changed it to `system.github_oauth_token` as `user` scoping does not apply to action aliases.